server/middlewares/validation.js

```diff
@@ -165,7 +165,7 @@ export const checkReviewsInput = (req, res, next) => {
     comments: {
       notEmpty: true,
       isLength: {
-        options: [{ min: 5, max: 250 }],
+        options: [{ min: 5, max: 500 }],
         errorMessage: reviewError
       },
       errorMessage: 'Your Business name is required'
```

Comment:
Good work so far. However, this seems to be a validation for reviews, if it requires a business name I assume it could be any business so saying your business name is required can be a bit confusing you can use something like Business name is required instead. Please do try to refactor the errorMessage. cheers

Reply:
Thank you for the feedback @Daramola98 would refactor this ASAP